### PR TITLE
feat: Add LandingPage articles to search results

### PR DESCRIPTION
**Pull request by jcbcapps**
# SC-3010

## Proposed changes
- Return articles with a category of `LandingPage` in search results
- `LandingPage` articles are given the proper permalink to link them to their associated landing page
- Articles with a category of `ORBITBlog` are also now returned in search results

---

## Reviewer notes
To test:
- Create a Tag
- Create an article with the category `LandingPage` that uses the tag created in the last step
- Create a Landing Page that uses the same tag
- Once everything is published, navigate to the portal and search for your article. It should appear in the search results and have a path that relates it to the landing page. Example path: `http://localhost:3000/landing/test-page/test-article`

Something to be aware of: if an article is linked to more than one landing page, only the first connection that is found will be returned in the search results.

E2E tests: https://github.com/USSF-ORBIT/ussf-portal/pull/357

### Start the system
```sh
yarn services:up
yarn dev
cd ../ussf-portal-client
yarn dev
```

Login to the cms http://localhost:3001

---

### As the original developer, I have

- [ ] Met the acceptance criteria
- [ ] Created new stories in Storybook if applicable
- [ ] Created/modified automated unit tests in Jest
- [ ] Created/modified automated [E2E tests](https://github.com/USSF-ORBIT/ussf-portal)
- [ ] Followed guidelines for zero-downtime deploys, if applicable
- [ ] Use [ANDI](https://www.ssa.gov/accessibility/andi/help/install.html) to check for basic a11y issues

### As a reviewer, I have

Check out our [How to review a pull request](https://github.com/USSF-ORBIT/ussf-portal/blob/main/docs/how-to/review-pull-request.md) document.

---

### CODE REVIEW

File: src/lib/schema.ts [hunk 1]
```diff
@@ -187,9 +187,30 @@ export const extendGraphqlSchema = (baseSchema: GraphQLSchema) =>
                   await prisma.article.findMany({
                     ...articleQuery,
                   })
-                ).map((article: ArticleQueryResult) => {
-                  // TODO: need to handle landing page articles differently here
-                  const permalink = `${process.env.PORTAL_URL}/articles/${article.slug}`
+                ).map(async (article: ArticleQueryResult) => {
```
Comment (gidjin) [inline src/lib/schema.ts:190]: Question: Is there a way to add a unit test to `src/lib/schema.test.ts` for this? Or at least a e2e test maybe? Maybe there is an e2e test coming but I didn't see a PR for it yet.

**Review by gidjin — COMMENTED** [review]
Looks good and works for me. Left a question about tests.

File: src/lib/schema.ts (hunk 1 above)
Comment (jcbcapps) [reply src/lib/schema.ts:190]: @gidjin I actually opted to add an e2e test: https://github.com/USSF-ORBIT/ussf-portal/pull/357

**Review by gidjin — APPROVED** [review]
LGTM!

**Pull request merged**